### PR TITLE
Update Styles

### PR DESCRIPTION
**Pull request by will-byrne**

### CODE REVIEW

File: client/src/core/styles/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,157 @@
+//body text
+.typography__body {
```
Comment (Cooryd) [inline client/src/core/styles/_typography.scss]: Weren't we going to put this in some kind of normalize stylesheet and apply this directly to the  `body` element?

File: client/src/core/styles/_typography.scss [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+}
+
+//headings
+.typography__heading {
+  color: $colourText;
+  font-weight: 600;
+  font-family: $fontStrong;
+}
+
+.typography__heading--h1 {
```
Comment (Cooryd) [inline client/src/core/styles/_typography.scss]: Should these be mixins applied to the `h` elements in a normalize stylesheet?

Comment (will-byrne) [reply client/src/core/styles/_typography.scss]: These should be classes if the idea is we can reuse them on non h tags. while we could use the mixin for that element it means adding another class which seems a bit redundant.

File: client/src/core/styles/_typography.scss (hunk 1 above)
Comment (will-byrne) [reply client/src/core/styles/_typography.scss]: I think that is something we can look into but is not really in the scope of this change. This was only intended to be a quick update based on the things that have changed for the login page.

**Pull request merged**